static/css/style.css

```diff
@@ -6849,3 +6849,384 @@ ul.submit-options li:last-of-type {
+	text-align: left;
+}
+
+.dei-at-storj-btn {
+	float: left;
+}
+
+.dei-blm {
+	background: #000;
+	margin-bottom: 30px;
+}
+
+@media only screen and (max-width: 991px) {
+	.dei-blm {
+		width: 95%;
+		margin: 0 auto;
+		margin-top: 30px;
+	}
+}
+
+.dei-blm-content {
+	max-width: 650px;
+	margin: 0 auto;
+}
+
+@media only screen and (max-width: 991px) {
+	.dei-blm-content {
+		width: 80%;
+	}
+}
+
+.dei-blm-heading {
+	font-family: Poppins;
+	font-size: 28px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 41px;
+	letter-spacing: 0px;
+	text-align: center;
+	margin-bottom: 0;
+}
+
+.dei-blm-subheading {
+	font-family: Roboto;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 300;
+	line-height: 21px;
+	letter-spacing: 0px;
+	text-align: center;
+}
+
+.dei-blm-btn {
+	color: #000;
+	background: #fff;
+}
+
+.dei-blm-btn:hover {
+	background: #ddd;
+	color: #000;
+}
+
+.dei-commitment {
+	background: #F4F6F9;
+}
+
+.dei-commitment-heading {
+	font-family: Poppins;
+	font-size: 28px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 41px;
+	letter-spacing: 0px;
+	text-align: center;
+}
+
+.dei-commitment-title {
+	font-family: Poppins;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 25px;
+	letter-spacing: 0px;
+	text-align: left;
+	min-height: 50px;
+}
+
+@media only screen and (max-width: 991px) {
+	.dei-commitment-title {
+		min-height: 30px;
+	}
+}
+
+@media only screen and (max-width: 991px) {
+	.dei-commitment-col {
+		margin-bottom: 50px;
+	}
+}
+
+@media only screen and (max-width: 767px) {
+	.dei-commitment-col {
+		margin-bottom: 90px;
+	}
+}
+
+
+.dei-commitment-content {
+	font-family: Roboto;
+	font-size: 14px;
+	font-style: normal;
+	font-weight: 300;
+	line-height: 20px;
+	letter-spacing: 0px;
+	text-align: left;
+}
+
+.dei-commitment-link {
+	font-family: Poppins;
+	font-size: 12px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 26px;
+	letter-spacing: 0px;
+	float: left;
+	color: #0055C6;
+}
+
+.dei-commitment-carrot {
+  position: relative;
+  left: 3px;
+	bottom: 1px;
+}
+
+.dei-principle-heading {
+	font-family: Poppins;
+	font-size: 28px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 41px;
+	letter-spacing: 0px;
+	text-align: center;
+}
+
+.dei-principle-col {
+	background: #F4F6F9;
+	text-align: left;
+	padding: 10px 30px 30px;
+  height: 392px;
+}
+
+@media only screen and (max-width: 991px) {
+	.dei-principle-col {
+		margin-bottom: 30px;
+		height: auto;
+	}
+}
+
+.dei-principle-img {
+	margin: 30px 0 10px 0;
+	height: 30px;
+}
+
+.dei-principle-title {
+	font-family: Poppins;
+	font-size: 22px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 41px;
+	letter-spacing: 0px;
+	text-align: left;
+}
+
+.dei-principle-content {
+	font-family: Roboto;
+	font-size: 16px;
+	font-style: normal;
+	font-weight: 300;
+	line-height: 24px;
+	letter-spacing: 0px;
+	text-align: left;
+}
+
+.dei-values {
+	background: #0055C6;
+	margin-top: 30px;
```

Comment:
Based on the design, the border around this content is on all sides, so replace with margin: 20px;